runtime/gc_glue_java/ConfigurationDelegate.hpp

```diff
@@ -71,6 +72,66 @@ class MM_ConfigurationDelegate
  * Member functions
  */
 private:
+	MM_HeapRegionDescriptorStandardExtension *
+	allocateHeapRegionDescriptorExtension(MM_EnvironmentBase* env, uintptr_t listCount) {
```

Comment:
fix position of '*' (should be next to the variable, not type) throughout the code you've touched

Reply:
Fixed my changes + other exiting code in the file